src/components/molecule/pageDescription/PageDescription.tsx

```diff
@@ -0,0 +1,34 @@
+import TextBox from '@/components/atom/TextBox';
+import React from 'react';
+
+interface DescriptionType {
+  title?: string;
+  subTitle?: string;
+  content?: string;
+  compo?: React.ReactNode;
+}
+
+const PageDescription = ({
+  title,
+  subTitle,
+  content,
+  compo,
+}: DescriptionType) => {
+  return (
+    <div className='bg-primary-white'>
+      {compo ? (
+        compo
+      ) : (
+        <div className='px-[16px] py-[40px] flex flex-col gap-[40px]'>
```

Comment:
```suggestion
        <div className='px-4 py-10 flex flex-col gap-10'>
```
현재 `tailwind.config.ts` 에서 `spacing` 값을 별도로 설정하지 않았는데, `px-[16px`], `py-[40px]` , `gap-[40px]` 처럼 `[숫자]` 표기법을 사용하신 이유가 있을까요?!

Tailwind는 기본적으로 `px-4(16px)`, `py-10(40px)`, `gap-10(40px)` 같은 값을 제공하기 때문에
특별한 이유가 없다면 기본 클래스를 사용하는 것이 더 일관된 코드 스타일을 유지하는 데 좋을 것 같습니다!

[Tailwind 공식 문서](https://v3.tailwindcss.com/docs/padding)